libcontainer/intelrdt/intelrdt.go

```diff
@@ -509,7 +509,7 @@ func (m *Manager) Destroy() error {
 	// Don't remove resctrl group if closid has been explicitly specified. The
 	// group is likely externally managed, i.e. by some other entity than us.
 	// There are probably other containers/tasks sharing the same group.
-	if m.config.IntelRdt == nil || m.config.IntelRdt.ClosID == "" {
+	if m.config.IntelRdt != nil && m.config.IntelRdt.ClosID == "" {
```

Comment:
In a view of the commit that follows this one, the code here can be simplified to
```go
m.config.IntelRdt.ClosID == "" {
```
(surely this also means this commit should be placed after)